homeassistant/components/rmvtransport/sensor.py

```diff
@@ -156,17 +155,42 @@ def state(self):
     def state_attributes(self):
         """Return the state attributes."""
         try:
-            return {
+            attrs = {
                 "next_departures": [val for val in self.data.departures[1:]],
                 "direction": self.data.departures[0].get("direction"),
                 "line": self.data.departures[0].get("line"),
                 "minutes": self.data.departures[0].get("minutes"),
                 "departure_time": self.data.departures[0].get("departure_time"),
                 "product": self.data.departures[0].get("product"),
             }
+            on_map = ATTR_LATITUDE, ATTR_LONGITUDE
+            no_map = "lat", "long"
+            lat_format, lon_format = on_map if self._show_on_map else no_map
+            try:
+                attrs[lon_format] = self.data.station_info["long"]
+                attrs[lat_format] = self.data.station_info["lat"]
+            except (KeyError, TypeError):
+                pass
+
+            return attrs
         except IndexError:
             return {}
 
+    @property
+    def device_info(self):
+        """Return a device description for device registry."""
+        return {
+            "next_departures": [val for val in self.data.departures[1:]],
```

Comment:
I don't think we can set any info as device_info.

https://developers.home-assistant.io/docs/en/device_registry_index.html#device-properties